README.md

```diff
@@ -1,2 +1,75 @@
-# bootstrap
-Foundation Bootstrap Team
+# Node+JS Foundation Bootstrap
+
+This is the operating repository for the Node+JS Bootstrap Team.
+
+The Node+JS Bootstrap Committee is the temporary body tasked with managing community recommendations for the transition of the merged Node+JS Foundation to a permanent governance model.
+
+The files in this repository are living documents and will never represent finished governance. Once we reach consensus on governance, this repository will be archived and the documents will be moved to their new home in a foundation specific Github org.
+
+Until such a time that this group has consensus on a new governance model, and the Foundation has legal squared away, all projects, programs, and committees will continue to operate unchanged.
+
+Upon enactment of this new governance model ("Ignition"), this working body will be disbanded.
+
+> Note: If there is to be an extended Bootstrap transition committee to, for example, provide oversight in the organization's first year, those criteria must be defined as a new program in the proposed governance model. There is no extension of **this** specific group's life-cycle.
+
+## Definitions
+Throughout this repository we may use assorted acronyms and jargon to describe many complex concepts. This language is subject to change as we iterate on proposals. To help onboard and level-set, we will maintain an active list of commonly used language here. If you see domain specific language used in this repository that is not represented here, please add it!
+
+| Vocabulary | Definition |
+|---|---|
+| Bootstrap | This repo, team, and process. |
+| Ignition | The decision to move forward with the new governance |
+| Contributor | You, and anyone that interacts with this repo. |
+| TLCs | The Top Level Councils of the project – Currently the CPC and C3 (defined below) |
+| CPC | A new proposed Top Level Committee – The "Cross Project Council |
+| C3 | A new proposed Top Level Committee – The "Cross Community Council |
+| CNCF | The Cloud Native Computing Foundation (Prior Art) |
+| COC | Code of Conduct |
+| MVF | Minimum Viable Foundation. The bare minimum governance documentation to operate a function joint foundation |
+| RFC | Request For Comments. The feedback and iteration process followed in the `/programs` directory. |
+
+## Deliverables
+The deliverables from this body will be:
+
+  - [ ] A recommendations document to help inform the new Foundation's by-laws (Goal: November 1st)
+  - [ ] A recommendations document to help inform the TLCs' charters (Goal: ???)
+  - [ ] A collection of fully drafted governance documents required for Ignition. "Minimum Viable Foundation" (MVF). (Goal: March 1st)
+  - [ ] A collection of program, initiative, and governance recommendation documents not required for MVF. After Ignition these will be passed to the appropriate bodies (CPC, C3, etc.) to execute.
+
+## Operations
```

Comment:
TLDR; I think we should remove everything below this line and punt to https://github.com/nodejs/bootstrap/pull/19